packages/react-composites/src/composites/CallComposite/components/ComplianceBanner.tsx

```diff
@@ -98,6 +100,7 @@ export const ComplianceBanner = (props: ComplianceBannerProps): JSX.Element => {
     <></>
   ) : (
     <MessageBar
+      className={mergeStyles(complianceBannerWarningStyles(theme))}
```

Comment:
Fluent's message bar has a `messageBarType={MessageBarType.Warning}` that sets color (though potentially other things) - can we use this instead?

Reply:
https://developer.microsoft.com/en-us/fluentui#/controls/web/messagebar

Reply:
The problem is the default colors they choose in those warnings can be similar to the video gallery background which is why we want to align with our fluent theme palette instead

Reply:
using the semanticColours here does guarantee that it matches Alex's designs. but I can look into what happens when its set to type warning as well, is the concern that when in the dark theme the warning colour could be different in this case?

Reply:
Looking into this both Light and dark themes with James' suggestion do result in the correct colour. do we want to go with that? or should we stick with the new style added?

Reply:
Better to go with the inbuilt prop that custom overrides (it may also have non-obvious bonuses like a11y related props set)